static/docs/use-cases/using-shared-server-for-the-team.md

```diff
@@ -0,0 +1,94 @@
+# Using Shared Server For The Team
+
+The key principle that every data science team should aim for is to make
+everything easilly reproducibile. Git makes it easy to share the code that you
+use to process your data and share it with the others. DVC helps with building
+your data pipelines and making them easilly reproducible. One of the best
+practices for data science teams is to work off the same data. Sharing your
+data is super valuable as it enables your colleagues and you to take advantage
+of the data processing that you've already done, and understand the context
+behind results you create and surface potential caveats. In order to share
+common data, many teams setup a shared server for the whole team. With DVC,
```

Comment:
May be we should omit or simplify the description above? It's too long, I'm not sure it's to the point of this use case.

Reply:
Ok